tests/function/test_cli.py

```diff
@@ -177,3 +180,98 @@ def test_config(self, virtwho, ssh_host):
         assert (result['send'] == 1
                 and result['error'] == 0
                 and msg in result['log'])
+
+    @pytest.mark.tier1
+    def test_kill_virtwho_in_terminal_side(self, virtwho, ssh_host):
+        """Test virt-who cli can be killed in terminal side by 'kill -2'
+
+        :title: virt-who: cli: test kill virt-who in terminal
+        :id: 9ed38e14-87dc-45b5-8656-b7933f6c72f2
+        :caseimportance: High
+        :tags: tier1
+        :customerscenario: false
+        :upstream: no
+        :steps:
+
+            1. start a thread to run virt-who by cli
+            2. kill the virt-who by 'kill 2'
+
+        :expectedresults:
+
+            1. virt-who cli can be killed normally in terminal side
+        """
+        t1 = threading.Thread(target=virtwho.run_cli(oneshot=False))
+        t1.start()
+        time.sleep(15)
+        assert virtwho.thread_number() == 1
+
+        # kill virt-who by 'kill -2'
+        _, _ = ssh_host.runcmd("ps -ef |"
```

Comment:
I think no need to use `_, _` if we don't need the output for this function, consider using it directly:
`ssh_host.runcmd("ps -ef |"
...`

Reply:
yes, will perfect it for some cases in different test files in another pr.